static/site.js

```diff
@@ -1,4 +1,6 @@
 (function () {
+  const loginLink = document.getElementsByClassName('js-login-link')[0];
```

Comment:
Using `id` instead of `class` would be because there can only be one element with a given `id`, so then getting that one element by `id` is easier than getting one element by `class`?

Although `id` would work (and even be more convenient) in this particular case, I think using `class` works more generally, because if I want to attach another `js-something` controller to this same element then I can just add another class for it, whereas I can't add a second `id`. Things like [the Trello CSS guide](https://github.com/trello/trellisheets/blob/master/styleguide.md#3-javascript) always seem to use classes for this.

This does make me think that the class may be misnamed though. Rather than `"js-login-link"` it should probably be something like `"js-open-login-popup"`. The js class should be named after the js controller/behavior that it attaches (open the login popup window on click) rather than after the HTML element that it happens to be on. Then you could use `"js-open-login-popup"` in multiple places (if you changed the js code to find and attach to all instances of that class).